examples/golang/README.md

```diff
@@ -0,0 +1,42 @@
+# Golang Examples
+
+This folder contains examples that can be run with [wasmcloud][wasmcloud] which are written in [Golang][golang].
+
+Most examples compile with [TinyGo][tinygo] rather than the Golang compiler proper given that WebAssembly support landed in TinyGo first and is still slightly better supported there.
```

Comment:
this `tinygo` link didn't resolve properly

Reply:
Oh good catch, fixed!